templates/create-daml-app/README.md

```diff
@@ -0,0 +1,132 @@
+**DISCLAIMER: This is a work in progress!**
```

Comment:
I thought we want to make this the default in 1.0? Shouldn’t we remove this comment in that case?